File: src/backend/app/googledetail.js
var keyid = "AIzaSyBxEIXyiNI3s4lDH6hMlC2JKsNGioD7iFg";

module.exports = function(app){

  app.get('/google/detail', function(req, res){

    var url = "https://maps.googleapis.com/maps/api/place/details/json?placeid=" + req.query.placeid + "&key=" + keyid;

    var https = require('https');

    https.get(url, function(response){
      var body = "";

      response.on('data', function(chunk){
        body += chunk;
      });
      response.on('end', function(){
        res.json(body);
      });
    });
  });
}

File: src/backend/googledetail.py
import requests as req 
 
 
 
def main(t):
    keyid = "AIzaSyCT_1vO2SmSjSFKTpsPJTvR1flLjSOpC-w"
    url = "https://maps.googleapis.com/maps/api/place/details/json?placeid="\
    + str(t) + "&key=" \
    + keyid
    r=req.get(url)
    return r.text
 
 
if __name__=="__main__":
    main() 
 
